xml/Microsoft.VisualBasic/Conversion.xml

```diff
@@ -335,8 +335,13 @@ End Class
   
  `Fix(number)` is equivalent to `Sign(number) * Int(Abs(number)).`  
   
+Starting with Visual Basic 15.8, the performance of decimal-to-integer conversion is optimized if you pass the value returned by the `Fix` method to the [CInt](~/docs/visual-basic/language-reference/functions/conversion-functions.md) function. For example:
```

Comment:
Somewhere, we should clarify the impact of this optimization. Is it faster? By how much?

Reply:
Faster. 

@DualBrain did some tests when he mentioned to me (I think over the phone). I don't recall the numbers. 

Do you think "significantly faster" is sufficient here?

Reply:
Depending on the tests, it can be up to twice as fast.  Of course, this is *only* noticed if you are doing a ton of these operations as twice as fast as a little bit slow really isn't too noticeable except in the most specific of scenarios... in those scenarios... when it's noticed, it's noticed. ;-)

Although I didn't go into the performance differences... I did do a "thank you" (with some details) post over at http://addressof.com/blog/archive/2018/09/20/Performance-Improvement-_2D00_-CInt_2800_Fix_2800_value_29002900_.aspx

Reply:
Ron, can we say something like "This optimization allows code to run faster - up to twice as fast for code that does a lot of conversion to integer types."